File: backend/src/main/java/com/banksphere/onboarding/dto/response/ApplicationDetailsResponse.java
package com.banksphere.onboarding.dto.response;

import com.banksphere.onboarding.enums.ApplicationStatus;

import java.time.LocalDate;
import java.time.LocalDateTime;

public record ApplicationDetailsResponse(String applicationReference, String firstName, String lastName, String email, String mobileNumber, String panNumber, String aadharNumber, ApplicationStatus status, LocalDateTime submittedAt , LocalDate dateOfBirth , Address address) {
}

File: frontend/src/features/officer/types/ApplicationDetailsResponse.ts
export interface ApplicationDetailsResponse {
  applicationReference: string;
  firstName: string;
  lastName: string;
  email: string;
  mobileNumber: string;
  panNumber: string;
  aadharNumber: string;
  status: "PENDING" | "APPROVED" | "REJECTED";
  submittedAt: string;
  dateOfBirth: string;
  address: {
    line1: string;
    line2: string;
    city: string;
    state: string;
    postalCode: string;
  };
}